<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterBien</class>
 <widget class="QDialog" name="AjouterBien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="VenteType">
      <item>
       <widget class="QLabel" name="investissementLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Type d'investissement</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="titreLabel">
          <property name="text">
           <string>Titre           </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titreLineEdit"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="typeInvestissementLayout">
        <item>
         <widget class="QCheckBox" name="achatButton">
          <property name="text">
           <string>Achat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="locationButton">
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="typeBienLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Type de bien</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="typeBienLayout">
        <item>
         <widget class="QRadioButton" name="maisonRadioButton">
          <property name="text">
           <string>Maison</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="appartementRadioButton">
          <property name="text">
           <string>Appartement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="chateauRadioButton">
          <property name="text">
           <string>Château</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fermeRadioButton">
          <property name="text">
           <string>Ferme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bureauRadioButton">
          <property name="text">
           <string>Bureau</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="commerceRadioButton">
          <property name="text">
           <string>Commerce</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="autreRadioButton">
          <property name="text">
           <string>Autre</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="nbPiecesLayout">
        <item>
         <widget class="QLabel" name="nbPiecesLabel">
          <property name="text">
           <string>Nombre de pièces</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="nbPiecesSpinBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="superficieLayout">
        <item>
         <widget class="QLabel" name="superficieLabel">
          <property name="text">
           <string>Superficie(m²)      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="superficieSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="habitableLabel">
          <property name="text">
           <string>                                                  Dont habitable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="habitableSpinBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="adresseLayout">
        <item>
         <widget class="QLabel" name="adresseLabel">
          <property name="text">
           <string>Adresse     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="adresseTextEdit">
          <property name="maximumBlockCount">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="descriptionLayout">
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="descriptionTextEdit"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="prixLayout">
      <item>
       <widget class="QLabel" name="prixLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="enVentelabel">
          <property name="text">
           <string>A la vente  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="prixVenteSpinBox">
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="euroLabel">
          <property name="text">
           <string>€</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>                                                  A la location</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="prixLocationSpinBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="validationLayout">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="enregisterButton">
          <property name="text">
           <string>Enregistrer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="annulerButton">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
